.gitignore

```
@@ -16,6 +16,25 @@
 # Mono auto generated files
 mono_crash.*
 
+# CMake output
+out/
+game/
+CMakeFiles/
+cmake_install.cmake
+CMakeCache.txt
+CopyOfCMakeCache.txt
```

```suggestion
CopyOfCMakeCache.txt
ALL_BUILD.vcxproj
ALL_BUILD.vcxproj.filters
Northstar.sln
ZERO_CHECK.vcxproj
ZERO_CHECK.vcxproj.filters
loader_wsock32_proxy/loader_wsock32_proxy.vcxproj
loader_wsock32_proxy/loader_wsock32_proxy.vcxproj.filters
NorthstarDLL/NorthstarDLL.vcxproj
NorthstarDLL/NorthstarDLL.vcxproj.filters
```
those are all of the untracked files generated for me

---

Resolved in 7e0bc06dd62f5e9ee0075b90c00d4f44e1f1cfa3?

---

yes